easybuild/easyconfigs/c/cairomm/cairomm-1.14.4-GCC-11.3.0-cairo-1.17.4.eb

```diff
@@ -0,0 +1,39 @@
+# Updated to MesonNinja as the autogen.sh complained.
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'MesonNinja'
+
+name = 'cairomm'
+version = '1.14.4'
+versionsuffix = '-cairo-1.17.4'
+
+homepage = 'http://cairographics.org'
+description = "The Cairomm package provides a C++ interface to Cairo."
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['http://cairographics.org/releases/']
+sources = [SOURCE_TAR_XZ]
+checksums = ['4749d25a2b2ef67cc0c014caaf5c87fa46792fc4b3ede186fb0fc932d2055158']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
```

Comment:
Does it really need cmake?

Reply:
You were right, we do not need `cmake`. Apologies for that, I have removed it.